assets/javascript/balances.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+  const balanceToggle = document.getElementById('balance-toggle-button');
+  if (balanceToggle) {
+    console.log('Setting up balances');
+    const SHOW_LABEL = 'Show balances';
+    const HIDE_LABEL = 'Hide balances';
+    const PLACEHOLDER = '****';
+    balanceToggle.innerHTML = SHOW_LABEL;
+    const balances = document.getElementsByClassName('balance-amount');
+    for (let i = 0; i < balances.length; i++) {
+      balances[i].innerHTML = PLACEHOLDER;
+      balances[i].setAttribute('aria-hidden', true);
+    }
+    balanceToggle.onclick = e => {
+      const isHidden = e.target.innerHTML === HIDE_LABEL;
```

Comment:
Would it not be better to check the `aria-hidden` attribute? Feels better than checking the HTML.

Reply:
The `aria-hidden` is on the balances sections, not the button (`e.target`). Maybe we could store a local variable for state?

Reply:
Doh, sorry, nah leave as is in that case.